detox/test/android/app/src/main/java/com/example/NativeModule.java

```diff
@@ -113,6 +114,15 @@ public void crashMainThread() {
         });
     }
 
+    @ReactMethod
+    public void toggleNoneStorageSynchronization(Boolean enable) {
```

Comment:
What does this method name mean? Toggle all synchronization that is not storage? Or "none storage synchornization" is a term?

Reply:
The former. You find it confusing?

Reply:
Yes :) Maybe there is a typo, NoneStorage -> NonStorage.

Reply:
Right!!!